tests/integration/integration-tests.sh

```diff
@@ -113,9 +113,9 @@ setup_test_pod() {
 
   MY_KUBE_CONFIG=$(cat /home/${USER}/.kube/config)
```

Comment:
this needs to be adapted

Reply:
This actually refers to the local file system, not the one in the pod. It basically copies the local file system kube config into the pod